jsonschema-validation.md

```diff
@@ -295,7 +284,7 @@ the name of a property in the instance.
 
 Omitting this keyword has the same behavior as an empty object.
 
-## Vocabularies for Semantic Content With `format` {#format}
```

Comment:
There are considerable impactful changes for `format`.  Specifically, schema authors will no longer have the ability to specify that `format` should be validated.  Tool configuration is the only way to validate `format` when we remove vocabularies.

See [discussion](https://github.com/orgs/json-schema-org/discussions/724#discussioncomment-9527084).

Reply:
(Might not be an issue if #1520 goes through.)